dask_expr/reductions.py

```diff
@@ -176,7 +176,22 @@ def chunk_kwargs(self):
         return {"ignore_index": self.ignore_index, **self._subset_kwargs()}
 
     def _simplify_up(self, parent):
-        return
+        if self.subset is not None and isinstance(parent, Projection):
+            columns = parent.operand("columns")
+            if not isinstance(columns, list) and not hasattr(columns, "dtype"):
+                columns = [columns]
+            subset = self.subset
+            if not isinstance(subset, list) and not hasattr(subset, "dtype"):
+                subset = [subset]
```

Comment:
Should this be normalized somewhere else?  Maybe in the collection `drop_duplicates` code?

Reply:
Not sure, pandas can deal with both. Maybe it's easier not to worry about it there?

Reply:
Yes both are equivalent.

Ok sounds good, moved it over.

Reply:
We have to normalise the columns though, because scalar columns are a legal input